# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../cdparanoia/gcc34.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

The usual gcc-3.4 fixes.

  Rene Rebe <rene@exactcode.de>

--- cdparanoia-III-alpha9.8-orig/interface/utils.h	2000-04-20 00:41:04.000000000 +0200
+++ cdparanoia-III-alpha9.8/interface/utils.h	2004-05-03 21:29:26.000000000 +0200
@@ -112,6 +112,7 @@
       break;
     case CDDA_MESSAGE_FORGETIT:
     default:
+      ((void)0);
     }
   }
 }
@@ -127,6 +128,7 @@
       break;
     case CDDA_MESSAGE_FORGETIT:
     default:
+      ((void)0);
     }
   }
 }
@@ -169,6 +171,7 @@
       break;
     case CDDA_MESSAGE_FORGETIT:
     default:
+      ((void)0);
     }
   }
   if(malloced)free(buffer);
@@ -205,6 +208,7 @@
       break;
     case CDDA_MESSAGE_FORGETIT:
     default:
+      ((void)0);
     }
   }
   if(malloced)free(buffer);
